### PR TITLE
Port to ByteBuddy

### PR DESCRIPTION
**Pull request by safris**
This code includes all changes necessary to port the java-grizzly-ahc plugin to v1.0 of SpecialAgent.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <artifactId>opentracing-grizzly-ahc</artifactId>
     <version>0.0.1-SNAPSHOT</version>
 
-    <name>${project.groupId}:${project.artifactId}</name>
+    <name>SpecialAgent Rule for Grizzly AsyncHttpClient</name>
```
Comment (jam01) [inline pom.xml:11]: Can we keep the previous name?

File: pom.xml [hunk 2]
```diff
@@ -35,10 +35,10 @@
 
         <version.grizzly-http-client>1.15</version.grizzly-http-client>
         <version.opentracing-api>0.31.0</version.opentracing-api>
-        <version.specialagent-maven-plugin>0.0.1-SNAPSHOT</version.specialagent-maven-plugin>
+        <version.specialagent>1.0.1-SNAPSHOT</version.specialagent>
```
Comment (jam01) [inline pom.xml:38]: Shouldn't this be `1.0.1`?

File: pom.xml (hunk 1 above)
Comment (safris) [reply pom.xml:11]: This should be a descriptive name, because it is used in SEO.

File: pom.xml (hunk 2 above)
Comment (safris) [reply pom.xml:38]: Apologies, this should actually be `1.0.2-SNAPSHOT`.

File: pom.xml (hunk 1 above)
Comment (jam01) [reply pom.xml:11]: https://github.com/opentracing-contrib/java-jdbc/blob/master/pom.xml#L23
https://github.com/opentracing-contrib/java-grpc/blob/master/pom.xml#L23

I tried to follow what other projects had. Though I'm not opposed to name it differently I don't think the project is SpecialAgent specific, so I'm thinking it should not be part of the name.

File: pom.xml (hunk 2 above)
Comment (safris) [reply pom.xml:38]: This should now be `1.0.2`.

File: pom.xml (hunk 1 above)
Comment (safris) [reply pom.xml:11]: Yes, I agree. I believe the most proper name would be:
```xml
<name>OpenTracing Instrumentation Plugin for Grizzly AsyncHttpClient</name>
```

**Pull request merged**